src/components/Dashboard/Menu/Options.tsx

```diff
@@ -83,6 +80,10 @@ export const DashboardMenuOptions = () => {
     setOpenSection((prev) => (prev === label ? null : label))
   }
 
+  useEffect(() => {
+    setOpenSection(() => menuItems[0].label)
+  }, [])
+
```

Comment:
I don't think this is necessary, since you're already sending to the desired section.